templates/mail/notify.txt

```diff
@@ -1,22 +1,26 @@
 Hi cloud.gov user,
 
-cloud.gov frequently updates the programming language buildpacks available to our customers, and we recently updated 
```

Comment:
Should we lead with something like 'ACTION REQUIRED' to make it blatantly obvious that they need to do something, not just read this email?

Reply:
The subject line has "action required", which I think is adequate here. It's not actually _required_ by us - it's customer responsibility to decide what to do - so I suggest a light touch on the language for it.

Reply:
ok, I'm happy with the subject line containing 'action required'. 

They are going to pick up the changes at next application deploy right? Regardless of whether they restage now or not? So it's not optional? That is: they should probably figure this out now instead of waiting?

Reply:
They don't have to pick the changes if they specify otherwise in their buildpack, but generally the answer to this question would be good to include in the linked docs.